src/commands/GitHubCommand.ts

```diff
@@ -0,0 +1,52 @@
+import { Message, MessageEmbed } from "discord.js";
+import Command from "../abstracts/Command";
+import GitHubService from "../services/GitHubService";
+import { EMBED_COLOURS } from "../config.json";
+
+class GitHubCommand extends Command {
+	constructor() {
+		super(
+			"github",
+			"Shows the repo of the given user."
+		);
+	}
+
+	async run(message: Message, args: string[]) {
+		const embed = new MessageEmbed();
```

Comment:
Since to matter what happens you end up sending the embed, you might as well just do it once.